### PR TITLE
Feature/redux calendar

### PR DESCRIPTION
**Pull request by rickmansfield**
## Description

In the future mentor and mentee, events may need to be passed in redux. These blocks of code stub out the events in the action suite and set up a calendarReducer.js for the potential switch statement for various states of events. This code will make it easier for follow-up cohorts to plug and play and as templates for other events and tickets as needed. 

Fixes # ...provides stubs/templates in actions suite and calendarReducer that may be used or discarded as needed. 

## Type of change
- [x] New feature (non-breaking change which adds functionality)

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have removed unnecessary comments/console logs from my code
- [x] I have made corresponding changes to the documentation if necessary (optional)
- [x] My changes generate no new warnings
- [x] I have checked my code and corrected any misspellings
- [x] No duplicate code left within changed files
- [x] Size of pull request kept to a minimum
- [x] Pull request description clearly describes changes made & motivations for said changes

https://www.loom.com/share/c3be0eab04004d76a1288f601ab718da

### CODE REVIEW

**Comment by RaphaelHebert** [comment]
Great video that explains the changes very clearly. Code is easy to read. Good job.

**Review by Chris-Boyle-94 — APPROVED** [review]
This will definitely help future groups out a lot with managing state, great forward thinking

**Review by nhamilton1 — APPROVED** [review]
Awesome work, and a well-done loom video to go along with it!

**Review by teresafranxman97 — APPROVED** [review]
Great explanation of the process and really going over why and how this was implemented.

**Review by cyberkade — APPROVED** [review]
Aside from the odd issue that we discussed, everything looks great and I'd love to see this merged.

**Review by Retrofitt — APPROVED** [review]
Great loom video, walks us through nicely on what was implemented. The code is clean, very readable, great job!

File: src/state/reducers/index.js [hunk 1]
```diff
@@ -2,6 +2,12 @@
 // This allows for the simplification of flow when importing reducers into your actions throughout your app.
 import { combineReducers } from 'redux';
 
-const rootReducer = combineReducers({});
+import userReducer from './userReducer';
+import calendarReducer from './calendarReducer';
+
+const rootReducer = combineReducers({
+  user: userReducer,
+  calendarReducer,
```
Comment (ayv8er) [inline src/state/reducers/index.js:10]: Would it be better to replace line 10 for "calendar: calendarReducer," ?

**Review by ayv8er — APPROVED** [review]

**Review by Pluviio — APPROVED** [review]
I like the simplicity  of it all and accessibility this brings about

**Pull request merged**